tests/unit/Logger.js

```diff
@@ -113,7 +113,7 @@ describe('WereLogs Logger is usable:', () => {
         const logger = new Logger('test');
         assert.doesNotThrow(
             () => {
-                logger.setLevel('fatal');
+                logger.setLevel('error');
```

Comment:
In fact, it's a quick fix as current log dump level is error, hence setLevel('fatal') throws an error :)